Backendless/Service/UserService.cs

```diff
@@ -18,7 +18,32 @@ public class UserService
     private static string USER_MANAGER_SERVER_ALIAS = "com.backendless.services.users.UserService";
     private ILoginStorage _loginStorage = null;
 
-    public BackendlessUser CurrentUser{ get; set; }
+    private BackendlessUser _currentUser = null;
+
+    public BackendlessUser CurrentUser
+    {
+      get
+      {
+        if( _currentUser == null )
+        {
+          if(LoginStorage.HasData)
+          {
+            try
+            {
+              CurrentUser = Backendless.Data.Of<BackendlessUser>().FindById( LoginStorage.ObjectId );

```

Comment:
Can ObjectId be null at this point?

Reply:
Line 29 checks for the presence of data in the LoginStorage.